frontend/test/unit/specs/stores/ui-store.spec.js

```diff
@@ -101,11 +101,9 @@ describe("Ui Store", () => {
       for (const key of Object.keys(initialState)) {
         // isMobileUa is set to true only if we explicitly get a mobile UA
         // from cookie or the browser request
-        if (key === "isMobileUa") {
-          expect(uiStore[key]).toEqual(false)
-        } else {
-          expect(uiStore[key]).toEqual(initialState[key])
-        }
+        const exepectedStoreValue =
+          key === "isMobileUi" ? false : initialState[key]
+        expect(uiStore[key]).toEqual(exepectedStoreValue)
```

Comment:
This code is equivalent to the original but prevents the `no-conditional-expect` rule from needing to be disabled twice (or at all).